NAMESPACE

```diff
@@ -8,7 +8,6 @@ export(addOutputs)
 export(calculatePKAnalyses)
 export(clearOutputIntervals)
 export(clearOutputs)
-export(enum)

```

Comment:
@Yuri05 I am not exporting the enaum anymore. That means that I cannot run the tests . This fixes #206